cli.sh

```diff
@@ -1081,38 +1091,52 @@ ${SHARED_REPO_ITEM}"
 #   from the global or local list, or both.
 #####################################################
 clear_shared_hook_repos() {
-    CLEAR_GLOBAL_REPOS=
-    CLEAR_LOCAL_REPOS=
+    CLEAR_GLOBAL_REPOS=""
+    CLEAR_LOCAL_REPOS=""
+    CLEAR_SHARED_REPOS=""
+    CLEAR_REPOS_FAILED=""
 
     case "$1" in
-    "--global")
-        CLEAR_GLOBAL_REPOS=1
+    "--shared")
+        CLEAR_SHARED_REPOS=1
         ;;
     "--local")
         CLEAR_LOCAL_REPOS=1
         ;;
-    "--all")
+    "--global")
         CLEAR_GLOBAL_REPOS=1
+        ;;
+    "--all")
+        CLEAR_SHARED_REPOS=1
         CLEAR_LOCAL_REPOS=1
+        CLEAR_GLOBAL_REPOS=1
         ;;
     *)
-        echo "! One of the following must be used:" >&2
-        echo "    \$ git hooks shared clear --global" >&2
-        echo "    \$ git hooks shared clear --local" >&2
-        echo "    \$ git hooks shared clear --all" >&2
+        echo "! Unknown clear option \`$1\`" >&2
+        echo "  Usage: \`git hooks shared clear [--shared|--local|--global|--all]\`" >&2
         exit 1
         ;;
     esac
 
-    if [ -n "$CLEAR_GLOBAL_REPOS" ] && [ -n "$(git config --global --get githooks.shared)" ]; then
-        git config --global --unset githooks.shared &&
-            echo "Global shared hook repository list cleared" ||
+    if [ -n "$CLEAR_LOCAL_REPOS" ]; then
+        if ! is_running_in_git_repo_root; then
+            echo "! The current directory \`$(pwd)\` does not" >&2
+            echo "  seem to be the root of a Git repository!" >&2
             CLEAR_REPOS_FAILED=1
+        else
+            git config --local --unset githooks.shared
```

Comment:
The error case is not handled here anymore
```suggestion
            git config --local --unset githooks.shared || CLEAR_REPOS_FAILED=1
```

Reply:
I think that can never fail  in this case. Unsetting any variable which does not exist in a repo is `exit 0`... isn't it?

Reply:
But jeah, I added it back

Reply:
No I can't do ` || ...`  because git return exit code `5`, if its not existing etc.
So I think its fine we trust git to do the right thing...